content/docs/releases/v1.1.0.md

```diff
@@ -0,0 +1,84 @@
+short = "v1.1.0"
+weight = 1
+date = "2021-11-28"
++++
+
+NSM v1.1.1 is released via a set of [example use cases](https://github.com/networkservicemesh/deployments-k8s/tree/release/v1.1.1).
+
+## Kubernetes Compatibility
+NSM v1.1.1 has been successfully [integration tested](https://github.com/networkservicemesh/integration-k8s-kind/actions/runs/1514862151) with Kubernetes versions:
+
+- v1.22.1
+- v1.21.1
+- v1.20.7
+- v1.19.11
+- v1.18.15
+- v1.17.11
+
+## Known Issues
+
+###   GKE Compatibility
+GKE has manifest an odd Netlink IP Neighbor bug with IPv6 in its latest node-version.  This is expected to worked around automatically in NSM in the future.  Meanwhile, there is a workaround:
+- [NSM Bug](https://github.com/networkservicemesh/integration-k8s-gke/issues/215)
+- Workaround: [Downgrade to GKE Node version 1.20.10-gke.301](https://github.com/networkservicemesh/integration-k8s-gke/commit/c181a994e334582237ff1d01e40218dab076a3e0#diff-7fbc346a181ddc9afe0b046700089e48ad38eab607902568c516fcde9aa08354R4)
+
+## Note on v1.1.1 being first version
+Due to mechanical issues with the release machinery's interaction with GOPROXY, the first release of v1.1 is v1.1.1, not v1.1.0
+
+## Changes since v1.0.0
+
+### P2MP
+
+One of the most fundamental components of Network Service Mesh in Kubernetes is the Forwarder.  It handles the mechanics of cross connecting whatever 'local mechanism' (like kernel interface, vfio, or memif) is injected into the Client Pod to whatever local (kernel,vfio, memif) or remote (vxlan, wireguard, etc) mechanism is needed to reach the Network Service Endpoint providing its requested Network Service.
+
+Traditionally, the Forwarder has been Point to Point (P2P). P2P has the benefit of having a single simple contract: a packet(or frame) that goes in one side comes out the other.  
+
+As the number and variety of NSM use cases has grown, some use cases have emerged in which the Forwarder needs to do more complex cross connects than just P2P.  Point to multipoint (P2MP), by contrast to P2P, have a large number of mutually incompatible implementations.  Often P2MP will be spoken of in shorthand as if its is a single unitary thing: it is not.
+
+In order to support these sorts of use cases, NSM made a slight architectual enhancement: have the forwarder select the NSE (or in the case of P2MP NSEs) to which it should be connecting.  This change allows a Forwarder to support zero or more possible P2MP senarios.
+
+Given a Network Service that requires some particular P2MP capabilty (or any other kind of capability) from its Forwarder for one of its logical links, it is necessary to have some capacity of 'hinting' that requirement to the NSMgr so that it selects the correct Forwarder in the event that more than on is present.  NetworkService matches can now be decorated with 'metadata' with 'labels' for use by the NSMgr in selecting the correct Forwarder for each logical hop the match represents.
+
+
+### Interdomain improvements
```

Comment:
Hmm, I think we also can point at an example to get started to run interdomain.
Something like
```
(Try it)[https://github.com/networkservicemesh/deployments-k8s/tree/main/examples/interdomain#interdomain]
```

Reply:
Does this work? https://github.com/networkservicemesh/site/pull/207/commits/e159baebc8a6873441e2c052214e9aaff92c188c